src/array/metadata.rs

```diff
@@ -12,6 +12,7 @@ use crate::error::GeoArrowError;
 /// this value is omitted, edges will be interpreted as planar.
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 pub enum Edges {
+    /// Follow a spherical path rather than a planar.
```

Comment:
Maybe best to copy the text from here: https://github.com/geoarrow/geoarrow/blob/main/extension-types.md#extension-metadata (or link to it)

> A value of "spherical" instructs consumers that edges follow a spherical path rather than a planar one. If this value is omitted, edges will be interpreted as planar. This metadata key is only applicable to a geoarrow.linestring, geoarrow.polygon, geoarrow.multilinestring, geoarrow.multipolygon or geoarrow.wkb array and should be omitted for an Array with any other extension name.